.travis.yml

```diff
@@ -14,11 +14,11 @@ git:
   depth: 100
 install:
 - if [ $USE_MICROPYTHON = false ]; then pip install Pillow Sphinx sphinx_bootstrap_theme recommonmark evdev==1.0.0 && pip install -r ./docs/requirements.txt; fi
-- if [ $USE_MICROPYTHON = true ]; then ./.travis/install-micropython.sh && PYTHON=~/micropython/ports/unix/micropython; else PYTHON=python3; fi
+- if [ $USE_MICROPYTHON = true ]; then ./.travis/install-micropython.sh && MICROPYTHON=~/micropython/ports/unix/micropython; fi
 script:
 - chmod -R g+rw ./tests/fake-sys/devices/**/*
-- if [ $USE_MICROPYTHON = false ]; then $PYTHON -W ignore tests/api_tests.py; fi
-- if [ $USE_MICROPYTHON = true ]; then $PYTHON tests/api_tests.py; fi
+- if [ $USE_MICROPYTHON = false ]; then python3 -W ignore::ResourceWarning tests/api_tests.py; fi
```

Comment:
Can you update ``tests/README`` to have the same change? Come to think of it that README also needs a line for how to run the tests using micropython.